src/pages/instances/actions/InstanceBulkDelete.tsx

```diff
@@ -95,9 +96,22 @@ const InstanceBulkDelete: FC<Props> = ({ instances, onStart, onFinish }) => {
                   <br />
                 </>
               )}
-              Are you sure you want to delete <b>{deleteCount}</b>{" "}
-              {pluralizeInstance(deleteCount)}?{"\n"}This action cannot be
+              This will permanently delete delete <b>{deleteCount}</b>{" "}
+              {pluralizeInstance(deleteCount)}.{"\n"}This action cannot be
               undone, and can result in data loss.
+              <div className="affected-items">
```

Comment:
Same concern as for the bulk action above. Also, this should check all instances can be deleted, and we don't have a mix of delete/ignore instances.